.github/workflows/build.yml

```diff
@@ -26,17 +26,25 @@ jobs:
         config:
           - preset: ci-dev-client-only-qt5
             qt_version: "5.15"
+            tests: qt5
```

Comment:
While I don't like much of the idea of having a "duplicated" value to say what `qt_version` has, I would only argue that it would be more readable as `qt_version_major: 6` and also would add this for all configs.

Reply:
Well, the thing about adding it for all configs is, the intent is that unit tests will be skipped if `matrix.config.tests` isn't set at all. (Hence, also, the name.)

Reply:
right, I forgot you mentioned that, maybe `tests_with` is more meaningful?

Reply:
So it's basically a tristate: `qt5` == "Run the Qt5 tests", `qt6` == "Run the Qt6 tests", unset == "run no tests".

Reply:
I currently have it set only on the client-only builds, because I don't think any of the unit tests require the UI, do they? (If they do, I can add/move it to the client-and-ui builds.)

Reply:
Sorry, hadn't seen your reply, GitHub is being weird today. Sure, I'll change it to `tests_with`.

Reply:
iirc there are many unit tests that require ui

Reply:
Hm. Weird, they all pass in a client-only build!

Reply:
Possible the CMake config is auto-excluding the UI ones, though.

Reply:
^ Yup, it sure was. 49 tests with UI enabled, 14 without.

Reply:
So, I've now enabled tests for both client-only and client-and-ui builds. Though running them for client-only may be unnecessary, since they're just a subset of the tests that will all be run for client-and-ui.